### PR TITLE
Initialized the repo with cookiecutter data science structure

### PR DESCRIPTION
**Pull request by gcroci2**
I used [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/#starting-a-new-project) to create this project structure.

### CODE REVIEW

**Review by svenvanderburg — COMMENTED** [review]
Nice @gcroci2 ! I am a big fan of https://drivendata.github.io/cookiecutter-data-science/ there is a lot of good things in here. It is a very complete approach to structuring data science projects, maybe some of the bells and whistles might be overwhelming. I think the most important point is that it is good to have a standard project structure as a team/research lab. So It's good to force yourself to have standards and structure, but it is also important to deviate from this rigid structure if it doesn't make sense for you or the projects that you work on. 

I am curious how you guys there in Nijmegen would ideally structure a project, I am pretty sure you don't agree on everything that this cookiecutter proposes.

**Comment by gcroci2** [comment]
> Nice @gcroci2 ! I am a big fan of https://drivendata.github.io/cookiecutter-data-science/ there is a lot of good things in here. It is a very complete approach to structuring data science projects, maybe some of the bells and whistles might be overwhelming. I think the most important point is that it is good to have a standard project structure as a team/research lab. So It's good to force yourself to have standards and structure, but it is also important to deviate from this rigid structure if it doesn't make sense for you or the projects that you work on.
> 
> I am curious how you guys there in Nijmegen would ideally structure a project, I am pretty sure you don't agree on everything that this cookiecutter proposes.

Thanks for your comment @svenvanderburg ! I totally agree, and I have already removed some of the files that I am pretty sure we are not going to use for this application. Waiting for the others' feedback and ideas :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+
+## Repository structure
+
+I used [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/#starting-a-new-project) to create this project structure. The following is the directory structure:
+
+```
+├── LICENSE
+├── README.md          <- The top-level README for developers using this project.
+├── data
+│   ├── external       <- Data from third party sources.
```
Comment (DarioMarzella) [inline README.md:12]: ├── external or (raw)  <- BA and EL text data (csv)
├── external_processed <- csv files ready for the modelling
├── blast_dbs  <- Blast databases and MSAs to generate them
├── 3D_models
├── PSSMs
├── Mapped_PSSMs
├── Aligned_3D_models
├── HDF5
├── NN_models

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+│   ├── interim        <- Intermediate data that has been transformed.
+│   ├── processed      <- The final, canonical data sets for modeling.
+│   └── raw            <- The original, immutable data dump.
+│
+├── docs               <- For useful documentation
+│
+├── models             <- Trained and serialized models, model predictions, or model summaries
+│
+├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
```
Comment (DarioMarzella) [inline README.md:21]: Do we want to keep notebooks separate from the scripts? 
I would vote for keeping only scripts.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+│   │
+│   ├── features       <- Scripts to turn raw data into features for modeling
+│   │   └── build_features.py
+│   │
+│   ├── models         <- Scripts to train models and then use trained models to make
+│   │   │                 predictions
+│   │   ├── predict_model.py
+│   │   └── train_model.py
+│   │
+│   └── visualization  <- Scripts to create exploratory and results oriented visualizations
```
Comment (DarioMarzella) [inline README.md:47]: I would add also a "src/tools" for tools that are useful all over but have no specific place and a "misc" or "utils" folder under every subfolder of src (data, features, models).

File: README.md (hunk 1 above)
Comment (gcroci2) [reply README.md:12]: Seems great! I added some subfolders:
├── raw
├── blas_dbs
├── hdf5
├── 3d_models
  ├── aligned
  ├── unaligned
├── external
  ├── processed
  ├── unprocessed
├── pssm
  ├── mapped
  ├── unmapped

I think they are even clearer to distinguish in this way, but let me know if you like it. For NN_models, there is an entire folder in the main dir called `models`, if you mean train networks and serialized models. Is that good?

File: README.md (hunk 2 above)
Comment (gcroci2) [reply README.md:21]: It's usually useful to have interactive notebooks during the development, even if absolutely not necessary. One reason to keep them anyway (not all of us use notebooks, but some of us do) is that they can become tutorials for later user usage, so we'll probably need them at a certain point. So I would keep the folder, and If not all of us are going to use it for the development part, is perfectly fine.

File: README.md (hunk 3 above)
Comment (gcroci2) [reply README.md:47]: All added!

File: README.md (hunk 1 above)
Comment (svenvanderburg) [reply README.md:12]: Looks good! And regarding models: I think it is good to keep trained models and model predictions separate from the data, but as long as it is consistent I'm fine with anything that makes sense.

**Pull request merged**